addons/mail/static/src/core/common/out_of_focus_service.js

```diff
@@ -89,15 +89,15 @@ export class OutOfFocusService {
      * @param {string} [param0.icon] The icon to be displayed in the
      * notification.
      */
-    sendNotification({ message, sound = true, title, type, icon }) {
-        if (!this.canSendNativeNotification || !this.multiTab.isOnMainTab()) {
+    async sendNotification({ message, sound = true, title, type, icon }) {
+        if (!this.canSendNativeNotification || !(await this.mainTab.isOnMainTab())) {
             if (sound) {
-                this._playSound();
+                await this._playSound();
```

Comment:
I don't think we should bother with awaiting `_playSound`. Especially as current code resolves it when it starts playing, rather than when it ends, which is confusing either way.

Playing a sound is usually just a nice side-effect, I don't think any part of the code should ever wait on that.